css/products.css

```diff
@@ -96,4 +117,32 @@
   .my-3 {
     margin: 0;
   }
+
+  .grid-cols-auto {
+    grid-template-columns: 0;
+  }
+
+  .grid-col-1 {
```

Comment:
shouldn't `.grid-col-1` be `.grid-cols-1` to match the consistency of other classes `.grid-cols-auto` and `.grid-cols-2`

Reply:
`grid-col-1` is for creating one grid column, which is singular hence why it is so.

Reply:
okay